prow/pod-utils/gcs/metadata.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gcs
+
+// Started is used to mirror the started.json artifact.
+type Started struct {
+	// Timestamp is the time in epoch seconds recorded when started.json is created.
+	Timestamp int64 `json:"timestamp"`
+	// Node is the name of the node the job was run on.
+	Node string `json:"node"`
+	// Repos is deprecated.
+	Repos map[string]string `json:"repos"`
+	// Pull is deprecated.
+	Pull string `json:"pull"`
+	// Revision identifies the revision of the code the build tested.
+	// Revision can be either a SHA or a ref.
+	// TODO: resolve https://github.com/kubernetes/test-infra/issues/10359
+	Revision string `json:"revision"`
```

Comment:
I think we don't write revision in started.json (as of today)... (I'll update the gubernator PR)

Reply:
noted

Reply:
I think we can remove the revision field